packages/account-v2/src/components/base/WalletDropdown/WalletDropdown.scss

```diff
@@ -74,7 +74,7 @@
     label,
     &__field:focus ~ &__label {
         position: absolute;
-        top: -0.5rem;
```

Comment:
Can u recheck if there can be another way? Because base folder is a stop gap and it will be removed. So any changes u do in its CSS might get removed as well. Modifying the base components stylings might effect in all places it is used

Reply:
This was just a small change because I noticed that dropdown label was a bit out of place when a value was selected. I don't think we would need it when we use the dropdown from deriv ui.